dcmdata/tests/tchval.cc

```diff
@@ -154,7 +154,7 @@ OFTEST(dcmdata_checkStringValue)
 // maximum length cannot be checked if given in characters (and not bytes)
 //  CHECK_BAD ( "LO-07", DcmLongString::checkStringValueu("OFFIS e.V., Escherweg 2, 26121 Oldenburg, Germany, http://www.offis.de/", "1") )
   CHECK_GOOD( "LO-08", DcmLongString::checkStringValue("\\ _2_ \\ _3_ \\ _4_ \\ _5_ \\", "6") )
-  CHECK_GOOD( "LO-09", DcmLongString::checkStringValue("ESC\033aping", "1") )
+  CHECK_BAD( "LO-09", DcmLongString::checkStringValue("ESC only allowed for charset extension \033", "1") )
```

Comment:
I changed these tests, as the ESCAPE character is only allowed for the escape sequence, as per PS 3.5, 6.1.3:
> The ESC character shall be used only for ISO 2022 character set control sequences, in accordance with [Section 6.1.2.5](https://dicom.nema.org/dicom/2013/output/chtml/part05/chapter_6.html#sect_6.1.2.5).